07-routing-gossip.md

```diff
@@ -281,8 +281,19 @@ it wants to change fees.
 The creating node MUST set `signature` to the signature of the
 double-SHA256 of the entire remaining packet after `signature` using its own `node-id`.
 
-The creating node MUST set `short-channel-id` to
-match those in the already-sent `channel_announcement` message, and MUST set the least-significant bit of `flags` to 0 if the creating node is `node-id-1` in that message, otherwise 1.  It MUST set other bits of `flags` to zero.
+The flags bitfield is used to indicate the direction of the channel this update concerns, i.e., it identifies the node that this update originated from, and signal various options concerning the channel.
+The following table specifies the meaning of the individual bits:
+
+| Bit Position  | Name        | Meaning                          |
+| ------------- | ----------- | -------------------------------- |
+| 0             | `direction` | Direction this update refers to. |
+| 1             | `disable`   | Disable the channel.             |
+
```

Comment:
The table style matches the one in bolt 09-features, and also matches the Github Flavored markdown :wink: 

I moved the table and associated text out of the requirements section.